Formula/easy-rsa.rb

```diff
@@ -0,0 +1,54 @@
+class EasyRsa < Formula
+  desc "CLI utility to build and manage a PKI CA"
+  homepage "https://github.com/OpenVPN/easy-rsa"
+  url "https://github.com/OpenVPN/easy-rsa/archive/v3.0.8.tar.gz"
+  sha256 "fd6b67d867c3b8afd53efa2ca015477f6658a02323e1799432083472ac0dd200"
+  license "GPL-2.0-only"
+  head "https://github.com/OpenVPN/easy-rsa.git"
+
+  depends_on "openssl@1.1"
+
+  def install
+    # Rename and edit vars.example file to define easy-rsa variables for homebrew
+    mv "easyrsa3/vars.example", "easyrsa3/vars"
+    inreplace "easyrsa3/vars" do |s|
+      s.gsub! "#set_var EASYRSA\t\"${0%/*}\"",
+              "set_var EASYRSA\t\"#{etc}/pki\""
+      s.gsub! "#set_var EASYRSA_OPENSSL\t\"openssl\"",
+              "set_var EASYRSA_OPENSSL\t\"#{Formula["openssl@1.1"].bin}/openssl\""
+      s.gsub! "#set_var EASYRSA_PKI\t\t\"$PWD/pki\"",
+              "set_var EASYRSA_PKI\t\"$EASYRSA\""
+    end
```

Comment:
Can this option for configuration be submitted upstream so others can use it too?

Reply:
This is the [official configuration method](https://github.com/OpenVPN/easy-rsa/blob/master/easyrsa3/vars.example), sadly:

```
# HOW TO USE THIS FILE
#
# vars.example contains built-in examples to Easy-RSA settings. You MUST name
# this file 'vars' if you want it to be used as a configuration file. If you do
# not, it WILL NOT be automatically read when you call easyrsa commands.
#
# It is not necessary to use this config file unless you wish to change
# operational defaults. These defaults should be fine for many uses without the
# need to copy and edit the 'vars' file.
```